routing/result_interpretation_test.go

```diff
@@ -70,13 +86,67 @@ var resultTestCases = []resultTestCase{
 		failure:       lnwire.NewExpiryTooSoon(lnwire.ChannelUpdate{}),
 
 		expectedResult: &interpretedResult{
-			pairResults: map[DirectedNodePair]lnwire.MilliSatoshi{
-				getTestPair(0, 1): 0,
-				getTestPair(1, 0): 0,
-				getTestPair(1, 2): 0,
-				getTestPair(2, 1): 0,
-				getTestPair(2, 3): 0,
-				getTestPair(3, 2): 0,
+			pairResults: map[DirectedNodePair]pairResult{
```

Comment:
Shouldn't all nodes but the final have been marked as a success since they successfully forwarded?

Reply:
In case of `ExpiryTooSoon`, any of the nodes could have caused a delay triggering this failure.